org.openkj.OpenKJ.yml

```diff
@@ -0,0 +1,67 @@
+app-id: org.openkj.OpenKJ
+runtime: org.kde.Platform
+runtime-version: 5.15
+sdk: org.kde.Sdk
+command: openkj
+modules:
+  - codecs/liba52.json
+  - codecs/libmpeg2.json
+  - codecs/gst-plugins-ugly.json
+  - codecs/libass.json
+  - codecs/ffmpeg.json
+  - codecs/gst-libav.json
+  - codecs/gst-plugins-bad.json
```

Comment:
we usually don't split out modules like that.

Reply:
Sorry, I used another flathub av program with similar requirements as boilerplate.  I'll get them all merged into the main file.

Reply:
it's just style. I'm ok with keeping them, but some other reviewer might disagree.

Reply:
Wasn't a big deal to fix, thanks for letting me know :)